ledger/sandbox-on-x/src/main/scala/com/daml/ledger/sandbox/SandboxOnXForTest.scala

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.sandbox
+
+import com.daml.ledger.api.auth.{AuthService, AuthServiceWildcard}
+import com.daml.ledger.configuration.Configuration
+import com.daml.ledger.runner.common.MetricsConfig.MetricRegistryType
+import com.daml.ledger.runner.common.{
+  Config,
+  MetricsConfig,
+  ParticipantConfig => _ParticipantConfig,
+}
+import com.daml.lf.data.Ref
+import com.daml.lf.engine.{EngineConfig => _EngineConfig}
+import com.daml.lf.language.LanguageVersion
+import com.daml.platform.apiserver.ApiServerConfig
+import com.daml.platform.apiserver.SeedService.Seeding
+import com.daml.platform.configuration.{InitialLedgerConfiguration, PartyConfiguration}
+import com.daml.platform.indexer.IndexerConfig
+import com.daml.platform.usermanagement.UserManagementConfig
+import com.daml.ports.Port
+
+import java.time.Duration
+import java.util.UUID
+import scala.concurrent.duration._
+
+object SandboxOnXForTest {
+  val EngineConfig: _EngineConfig = _EngineConfig(
+    allowedLanguageVersions = LanguageVersion.StableVersions,
```

Comment:
It would be handy to be able to build a config against Stable/Dev without having to do those nested `copy` on `Default` you are doing in several place.

Reply:
I have tried now, please have a look ;)